plugins/analytics-provider-segment/package.json

```diff
@@ -10,11 +10,12 @@
     "types": "dist/index.d.ts"
   },
   "backstage": {
-    "role": "frontend-plugin-module"
+    "role": "frontend-plugin"
```

Comment:
why do we need to change role?

Reply:
I think the `frontend-plugin-module` is just a leftover. It's actually always treated the same as `frontend-plugin` by Backstage CLI.

https://github.com/backstage/backstage/blob/d141d12529687ad199523cca272947b43dae6e05/packages/cli/config/eslint-factory.js#L184-L185

https://github.com/backstage/backstage/blob/d141d12529687ad199523cca272947b43dae6e05/packages/cli/config/jest.js#L56-L57

https://github.com/backstage/backstage/blob/d141d12529687ad199523cca272947b43dae6e05/packages/cli/src/lib/bundler/packageDetection.ts#L86

However in Janus CLI, we (for the sake of simplicity) never actually considered it. So dynamic plugin export actually fails because of:

https://github.com/janus-idp/backstage-plugins/blob/30cc4b3f68e30c255fe8c02c2ec18450ee8b2e9c/packages/cli/src/commands/export-dynamic-plugin/command.ts#L52